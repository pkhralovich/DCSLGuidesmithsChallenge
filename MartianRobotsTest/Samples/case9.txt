5 5
5 0 N 5
FLRFLRFLRFLRFLRF