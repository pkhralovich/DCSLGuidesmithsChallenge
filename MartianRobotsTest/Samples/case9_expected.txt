Oops... Something happened: Invalid robot position format
